docs/reference/aggregations/bucket/geotilegrid-aggregation.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[search-aggregations-bucket-geotilegrid-aggregation]]
 === GeoTile Grid Aggregation
 
-A multi-bucket aggregation that works on `geo_point` fields and groups points into
+A multi-bucket aggregation that works on `geo_point` and `geo_shape` fields and groups points into
```

Comment:
"and groups points into buckets" leaves out how geo_shapes work. How about enhancing with text how geo_shapes are matched into buckets as well. Maybe something like "and groups points, lines, and polygons into buckets"?